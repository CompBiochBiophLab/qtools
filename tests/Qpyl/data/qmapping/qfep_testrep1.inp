21         # number of files/frames
2  0                 # number of states and predefined off-diagonals
0.5921868218 1        # RT and number of points to skip
20              # number of bins
1    # minimum points for bin
10.0         # state2 shift (alpha)
1                    # number of diagonal elements
1 2 100.0 0 0 0     # states 1 and 2, A=const=Hij, mu=eta=r0= 0
1 -1                 # linear combination of states ( E = e1 - e2 )
fep_000_1.000.en
fep_001_0.950.en
fep_002_0.900.en
fep_003_0.850.en
fep_004_0.800.en
fep_005_0.750.en
fep_006_0.700.en
fep_007_0.650.en
fep_008_0.600.en
fep_009_0.550.en
fep_010_0.500.en
fep_011_0.450.en
fep_012_0.400.en
fep_013_0.350.en
fep_014_0.300.en
fep_015_0.250.en
fep_016_0.200.en
fep_017_0.150.en
fep_018_0.100.en
fep_019_0.050.en
fep_020_0.000.en
stop