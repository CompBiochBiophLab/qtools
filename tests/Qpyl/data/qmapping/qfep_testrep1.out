
Build and version information

Build number 6.0.8
Build date   20170817
Built:       
      by     acmnpv
      on     acmnpvdell
      git id 7f54fcf7fd0dc0c14e640a3fa3ce03384d771623
      with   GNU Fortran Debian 6.4.0-1 6.4.0 20170704
qfep version 6.0.8 , git id=7f54fcf7fd0dc0c14e640a3fa3ce03384d771623 initialising


Welcome in qfep modification date 20170817

 Q6, Copyright © 2017 Johan Åqvist, John Marelius, Shina Caroline Lynn Kamerlin and Paul Bauer
 Q6 comes with ABSOLUTELY NO WARRANTY.  This is free software, and you are welcome
 to redistribute it under certain conditions. For details, add the --help flag.
--> Number of energy files: # Number of files                 =    21
--> No. of states, no. of predefined off-diag elements: # Number of states                 =     2
# Number of off-diagonal elements =     0
--> Give kT & no, of pts to skip   & calculation mode: # kT                              = 0.592
# Number of data points to skip   =     1
# Only QQ interactions will be considered   =   0
--> Give number of gap-bins: # Number of gap-bins              =    20
--> Give minimum # pts/bin: # Minimum number of points per bin=     1
--> Give alpha for state  2:                                                    # Alpha for state  2              = 10.00
--> Number of off-diagonal elements:# Number of off-diagonal elements =     1
#   i   j   A(i,j)  mu(i,j) eta(i,j) rxy0(i,j)
--> i, j, A_ij, mu_ij, eta_ij, r_xy0: #   1   2    100.00      0.00      0.00      0.00
--> linear combination of states defining reaction coord: # Linear combination co-efficients=  1.00 -1.00


# Part 0: Average energies for all states in all files
# file             state   pts   lambda    EQtot   EQbond  EQang   EQtor   EQimp    EQel   EQvdW  Eel_qq  EvdW_qq Eel_qp  EvdW_qp Eel_qw EvdW_qw Eqrstr
--> Name of file number   1:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_000_1.000.en        1         8  1.000000   -73.92   -65.80     0.54     0.05     0.00    -8.87    -0.17    -6.89    -0.17    -1.98    -0.01     0.00     0.00     0.34
fep_000_1.000.en        2         8  0.000000    99.55    -6.22    82.32     0.99     0.00    -7.70    30.15    -7.10    27.37    -0.60     2.78     0.00     0.00     0.00
--> Name of file number   2:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_001_0.950.en        1         8  0.950000   -74.73   -65.82     0.30     0.05     0.00    -9.33     0.03    -7.44     0.21    -1.89    -0.18     0.00     0.00     0.05
fep_001_0.950.en        2         8  0.050000    86.00    -9.63    74.86     1.08     0.00    -7.17    26.86    -6.53    24.78    -0.64     2.09     0.00     0.00     0.00
--> Name of file number   3:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_002_0.900.en        1         8  0.900000   -74.02   -65.42     0.50     0.08     0.00    -9.51     0.30    -7.69     0.57    -1.82    -0.27     0.00     0.00     0.03
fep_002_0.900.en        2         8  0.100000    78.96   -10.99    71.11     1.01     0.00    -7.30    25.13    -6.54    23.59    -0.76     1.53     0.00     0.00     0.00
--> Name of file number   4:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_003_0.850.en        1         8  0.850000   -72.79   -64.73     0.61     0.26     0.00    -9.19     0.14    -7.22     0.36    -1.97    -0.22     0.00     0.00     0.11
fep_003_0.850.en        2         8  0.150000    74.17    -8.47    67.33     0.77     0.00    -8.12    22.67    -7.32    21.61    -0.81     1.07     0.00     0.00     0.00
--> Name of file number   5:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_004_0.800.en        1         8  0.800000   -72.25   -64.40     0.70     0.19     0.00    -9.31     0.57    -7.49     0.82    -1.82    -0.24     0.00     0.00     0.00
fep_004_0.800.en        2         8  0.200000    65.41   -13.83    62.94     0.97     0.00    -7.45    22.78    -6.63    21.42    -0.82     1.35     0.00     0.00     0.00
--> Name of file number   6:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_005_0.750.en        1         8  0.750000   -70.41   -63.38     0.96     0.15     0.00    -9.19     1.05    -7.46     1.36    -1.74    -0.31     0.00     0.00     0.00
fep_005_0.750.en        2         8  0.250000    56.21   -16.26    57.76     1.00     0.00    -7.52    21.24    -6.73    20.15    -0.79     1.09     0.00     0.00     0.00
--> Name of file number   7:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_006_0.700.en        1         8  0.700000   -66.33   -61.33     1.87     0.22     0.00    -8.95     1.82    -7.16     2.06    -1.78    -0.24     0.00     0.00     0.03
fep_006_0.700.en        2         8  0.300000    48.24   -16.18    52.60     0.91     0.00    -8.33    19.23    -7.35    18.18    -0.98     1.05     0.00     0.00     0.00
--> Name of file number   8:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_007_0.650.en        1         8  0.650000   -63.52   -58.39     1.94     0.15     0.00    -8.98     1.76    -7.23     1.96    -1.75    -0.21     0.00     0.00     0.00
fep_007_0.650.en        2         8  0.350000    41.19   -17.64    50.66     1.03     0.00    -8.09    15.22    -7.15    14.16    -0.95     1.06     0.00     0.00     0.00
--> Name of file number   9:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_008_0.600.en        1         8  0.600000   -53.55   -55.25     5.37     0.24     0.00    -8.44     4.52    -6.88     4.81    -1.56    -0.29     0.00     0.00     0.00
fep_008_0.600.en        2         8  0.400000    18.58   -28.19    38.79     0.92     0.00    -8.27    15.33    -7.29    14.67    -0.98     0.65     0.00     0.00     0.00
--> Name of file number  10:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_009_0.550.en        1         8  0.550000   -38.75   -49.45    10.41     0.39     0.00    -7.51     7.42    -6.01     7.62    -1.50    -0.20     0.00     0.00     0.00
fep_009_0.550.en        2         8  0.450000    -2.46   -36.20    28.74     0.76     0.00    -8.85    13.09    -7.74    12.79    -1.11     0.30     0.00     0.00     0.00
--> Name of file number  11:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_010_0.500.en        1         8  0.500000   -18.11   -40.80    18.25     0.68     0.00    -6.21     9.97    -4.69     9.52    -1.52     0.45     0.00     0.00     0.00
fep_010_0.500.en        2         8  0.500000   -24.25   -43.48    18.52     0.52     0.00    -9.55     9.74    -8.30     9.45    -1.25     0.29     0.00     0.00     0.00
--> Name of file number  12:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_011_0.450.en        1         8  0.450000     9.79   -31.11    31.09     0.81     0.00    -4.37    13.37    -3.00    12.91    -1.37     0.46     0.00     0.00     0.00
fep_011_0.450.en        2         8  0.550000   -48.92   -54.98     9.16     0.36     0.00   -10.33     6.86    -8.99     6.93    -1.33    -0.07     0.00     0.00     0.00
--> Name of file number  13:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_012_0.400.en        1         8  0.400000    28.98   -25.24    40.26     0.88     0.00    -2.88    15.95    -1.59    15.37    -1.29     0.58     0.00     0.00     0.00
fep_012_0.400.en        2         8  0.600000   -62.57   -61.86     4.57     0.26     0.00   -10.68     5.14    -9.28     5.33    -1.40    -0.18     0.00     0.00     0.00
--> Name of file number  14:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_013_0.350.en        1         8  0.350000    51.79   -15.99    49.40     0.96     0.00    -1.99    19.41    -0.87    18.68    -1.12     0.73     0.00     0.00     0.00
fep_013_0.350.en        2         8  0.650000   -73.82   -67.63     2.06     0.19     0.00   -10.31     1.87    -8.82     2.10    -1.50    -0.24     0.00     0.00     0.00
--> Name of file number  15:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_014_0.300.en        1         8  0.300000    62.92   -12.23    54.15     0.95     0.00    -1.72    21.77    -0.54    20.97    -1.17     0.80     0.00     0.00     0.00
fep_014_0.300.en        2         8  0.700000   -77.08   -70.03     1.71     0.22     0.00    -9.98     0.97    -8.44     1.17    -1.54    -0.21     0.00     0.00     0.04
--> Name of file number  16:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_015_0.250.en        1         8  0.250000    72.44   -14.94    61.45     0.93     0.00    -0.08    25.09     1.01    24.12    -1.09     0.97     0.00     0.00     0.00
fep_015_0.250.en        2         8  0.750000   -81.19   -73.19     0.90     0.19     0.00   -10.70     1.61    -9.12     1.79    -1.57    -0.18     0.00     0.00     0.00
--> Name of file number  17:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_016_0.200.en        1         8  0.200000    82.67   -11.77    63.43     0.95     0.00     0.06    30.00     1.06    28.39    -1.00     1.62     0.00     0.00     0.00
fep_016_0.200.en        2         8  0.800000   -83.22   -74.70     0.66     0.18     0.00   -10.20     0.84    -8.58     0.94    -1.62    -0.10     0.00     0.00     0.01
--> Name of file number  18:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_017_0.150.en        1         8  0.150000    91.20    -8.14    67.09     0.95     0.00     0.11    31.19     1.03    29.60    -0.92     1.59     0.00     0.00     0.00
fep_017_0.150.en        2         8  0.850000   -84.38   -75.74     0.59     0.17     0.00    -9.89     0.38    -8.23     0.51    -1.66    -0.13     0.00     0.00     0.12
--> Name of file number  19:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_018_0.100.en        1         8  0.100000    98.96    -9.54    70.77     0.98     0.00     0.76    35.98     1.75    33.81    -0.98     2.17     0.00     0.00     0.00
fep_018_0.100.en        2         8  0.900000   -85.98   -76.90     0.49     0.10     0.00   -10.00     0.31    -8.31     0.41    -1.69    -0.09     0.00     0.00     0.02
--> Name of file number  20:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_019_0.050.en        1         8  0.050000   109.95    -9.06    76.71     0.87     0.00     1.48    39.95     2.51    37.63    -1.03     2.32     0.00     0.00     0.00
fep_019_0.050.en        2         8  0.950000   -87.00   -77.67     0.32     0.09     0.00   -10.19     0.42    -8.40     0.46    -1.79    -0.04     0.00     0.00     0.03
--> Name of file number  21:                                                     
# Energy files from Qdyn, version: 6.0.9 , git id=0670be695109be30d338c65b603bf6a779f78d51                         
fep_020_0.000.en        1         8  0.000000   126.07    -6.61    84.41     0.67     0.00     1.35    46.25     2.33    43.48    -0.99     2.77     0.00     0.00     0.00
fep_020_0.000.en        2         8  1.000000   -86.55   -77.87     0.42     0.10     0.00    -9.70     0.28    -7.59     0.14    -2.11     0.14     0.00     0.00     0.22


# Part 1: Free energy perturbation summary:

# Calculation for full system
# lambda(1)      dGf sum(dGf)      dGr sum(dGr)     <dG>
   1.000000     0.000     0.000    -8.569     7.024     0.000
   0.950000     9.086     9.086    -8.194    15.593     8.827
   0.900000     8.499    17.585    -7.887    23.787    17.174
   0.850000     8.100    25.686    -7.429    31.674    25.167
   0.800000     7.801    33.487    -6.946    39.103    32.783
   0.750000     7.342    40.829    -6.439    46.049    39.927
   0.700000     6.705    47.535    -5.964    52.488    46.499
   0.650000     5.972    53.507    -4.721    58.452    52.467
   0.600000     5.547    59.054    -3.643    63.173    57.601
   0.550000     3.388    62.442    -1.492    66.816    61.117
   0.500000     1.285    63.727     2.139    68.308    62.505
   0.450000    -0.983    62.744     3.880    66.168    60.944
   0.400000    -2.713    60.031     5.540    62.288    57.647
   0.350000    -4.210    55.821     6.270    56.748    52.772
   0.300000    -5.942    49.878     7.000    50.478    46.666
   0.250000    -6.770    43.108     7.545    43.478    39.781
   0.200000    -7.327    35.781     8.143    35.933    32.345
   0.150000    -8.042    27.739     8.620    27.790    24.252
   0.100000    -8.432    19.307     9.154    19.170    15.726
   0.050000    -8.837    10.470    10.015    10.015     6.731
   0.000000    -9.503     0.967     0.000     0.000    -3.029

# Min energy-gap is:   -201.63
# Max energy-gap is:    216.25



# Part 2: Reaction free energy summary:
# Lambda(1)  bin Energy gap      dGa     dGb     dGg    # pts    c1**2    c2**2
   1.000000    1    -191.18       0.00     182.72     -43.63      3     0.845     0.155
   1.000000    2    -170.28       0.00     178.13     -44.72      5     0.833     0.167
   0.950000    2    -170.28       0.26     164.38     -46.10      8     0.825     0.175
   0.900000    2    -170.28       0.50     166.00     -46.36      5     0.820     0.180
   0.900000    3    -149.39       1.45     158.31     -47.15      3     0.809     0.191
   0.850000    2    -170.28       0.93     161.52     -46.85      2     0.814     0.186
   0.850000    3    -149.39       1.59     152.06     -47.12      6     0.807     0.193
   0.800000    3    -149.39       2.56     147.80     -47.23      8     0.797     0.203
   0.750000    3    -149.39       3.83     146.76     -47.29      4     0.792     0.208
   0.750000    4    -128.50       6.78     134.64     -46.85      4     0.772     0.228
   0.700000    4    -128.50       6.15     131.17     -46.95      6     0.772     0.228
   0.700000    5    -107.60      13.51     121.50     -45.59      2     0.739     0.261
   0.650000    4    -128.50       6.05     132.74     -47.43      2     0.770     0.230
   0.650000    5    -107.60      11.97     119.35     -45.48      6     0.741     0.259
   0.600000    5    -107.60      14.89     118.43     -44.90      3     0.731     0.269
   0.600000    6     -86.71      23.81     108.29     -42.51      1     0.695     0.305
   0.600000    7     -65.82      28.05      98.42     -41.45      3     0.666     0.334
   0.600000    8     -44.92      38.24      86.65     -40.45      1     0.618     0.382
   0.550000    6     -86.71      20.10     104.72     -43.98      2     0.696     0.304
   0.550000    7     -65.82      27.37     102.36     -41.93      1     0.676     0.324
   0.550000    8     -44.92      43.55      82.59     -38.82      1     0.596     0.404
   0.550000    9     -24.03      48.39      74.14     -38.44      3     0.565     0.435
   0.550000   10      -3.13      57.74      65.24     -38.58      1     0.519     0.481
   0.500000    8     -44.92      39.87      85.07     -40.02      2     0.610     0.390
   0.500000    9     -24.03      53.46      71.55     -37.90      1     0.545     0.455
   0.500000   10      -3.13      57.97      64.15     -37.56      2     0.515     0.485
   0.500000   11      17.76      74.21      50.80     -38.18      1     0.442     0.558
   0.500000   12      38.65      79.14      45.83     -38.87      2     0.418     0.582
   0.450000   11      17.76      75.71      48.86     -38.61      1     0.433     0.567
   0.450000   12      38.65      83.83      39.62     -39.37      4     0.389     0.611
   0.450000   13      59.55      89.34      30.98     -40.85      3     0.356     0.644
   0.400000   13      59.55      95.24      32.58     -40.88      1     0.351     0.649
   0.400000   14      80.44     103.59      24.28     -42.25      5     0.312     0.688
   0.400000   15     101.34     112.69      20.95     -43.20      2     0.292     0.708
   0.350000   15     101.34     114.88      15.03     -44.44      2     0.273     0.727
   0.350000   16     122.23     127.10       8.99     -46.15      6     0.243     0.757
   0.300000   16     122.23     128.09       8.77     -46.17      5     0.240     0.760
   0.300000   17     143.12     143.79       2.79     -47.82      3     0.209     0.791
   0.250000   16     122.23     134.13       8.33     -46.91      1     0.234     0.766
   0.250000   17     143.12     145.42       2.58     -47.52      6     0.207     0.793
   0.250000   18     164.02     157.24       0.63     -48.08      1     0.192     0.808
   0.200000   17     143.12     145.63       3.73     -47.69      3     0.210     0.790
   0.200000   18     164.02     156.16      -1.27     -47.81      5     0.183     0.817
   0.150000   18     164.02     156.51      -0.65     -47.80      6     0.186     0.814
   0.150000   19     184.91     173.29      -2.42     -47.34      2     0.168     0.832
   0.100000   18     164.02     159.91      -1.20     -47.93      3     0.179     0.821
   0.100000   19     184.91     175.48      -2.30     -46.87      5     0.166     0.834
   0.050000   18     164.02     168.21      -1.77     -48.02      1     0.176     0.824
   0.050000   19     184.91     177.25      -2.75     -46.36      6     0.158     0.842
   0.050000   20     205.81     195.53      -3.21     -44.81      1     0.148     0.852
   0.000000   19     184.91     192.03      -3.03     -45.18      1     0.151     0.849
   0.000000   20     205.81     193.48      -3.03     -44.70      6     0.145     0.855


# Part 3: Bin-averaged summary:
# bin  energy gap  <dGg> <dGg norm> pts  <c1**2> <c2**2> <r_xy>
   1   -191.18    -43.63      4.22          3    0.845    0.155    1.934
   2   -170.28    -45.89      1.96         20    0.825    0.175    1.977
   3   -149.39    -47.20      0.66         21    0.800    0.200    2.045
   4   -128.50    -47.00      0.86         12    0.771    0.229    2.120
   5   -107.60    -45.34      2.51         11    0.738    0.262    2.197
   6    -86.71    -43.49      4.37          3    0.695    0.305    2.232
   7    -65.82    -41.57      6.29          4    0.668    0.332    2.311
   8    -44.92    -39.83      8.03          4    0.608    0.392    2.398
   9    -24.03    -38.30      9.55          4    0.560    0.440    2.481
  10     -3.13    -37.90      9.96          3    0.517    0.483    2.502
  11     17.76    -38.39      9.46          2    0.438    0.562    2.656
  12     38.65    -39.21      8.65          6    0.399    0.601    2.726
  13     59.55    -40.86      7.00          4    0.355    0.645    2.820
  14     80.44    -42.25      5.61          5    0.312    0.688    2.909
  15    101.34    -43.82      4.04          4    0.282    0.718    3.039
  16    122.23    -46.22      1.63         12    0.241    0.759    3.348
  17    143.12    -47.64      0.22         12    0.208    0.792    3.378
  18    164.02    -47.86      0.00         16    0.184    0.816    3.590
  19    184.91    -46.60      1.26         14    0.162    0.838    3.621
  20    205.81    -44.72      3.14          7    0.146    0.854    3.786
